tests/test_certificate_validator.py

```diff
@@ -150,3 +153,20 @@ async def test_self_signed_with_policy(self):
         self.assertEqual(qp.user_domain_policy_id, 'any_policy')
         self.assertEqual(qp.issuer_domain_policy_id, 'any_policy')
 
+    async def test_basic_certificate_validator_tls_aia(self):
```

Comment:
Any chance you could download these certs and commit them as part of the test suite? Apparently there's a bug in the macOS-specific part of the TLS handling in `oscrypto` causing the handshake to fail.

I'd also prefer only testing the fetcher here (and moving the test to the appropriate place for that). Given that this test depends on external data that we don't control, I want to keep the number of moving parts to the absolute minimum here :).